oracle/service.go

```diff
@@ -172,6 +184,16 @@ func NewService(
 		svc.pricePullers[ticker] = pricePuller
 	}
 
+	for _, feedCfg := range storkFeedConfigs {
+		ticker := feedCfg.Ticker
+		pricePuller, err := NewStorkPriceFeed(feedCfg)
+		if err != nil {
+			err = errors.Wrapf(err, "failed to init stork price feed for ticker %s", ticker)
+			return nil, err
+		}
+		svc.pricePullers[ticker] = pricePuller
```

Comment:
Warning: this will overwrite other dynamic feed configs if the ticker collides, I'm ok with it, as long as it is documented, maybe in the Readme so we know that Stork feeds in such case